src/main/resources/application.properties

```diff
@@ -1,29 +1,30 @@
+
 #for local db (swagger,postman etc)
 
-#spring.datasource.url=jdbc:h2:mem:demo;DB_CLOSE_ON_EXIT=FALSE
-#spring.datasource.username=sa
-#spring.datasource.password=
-#spring.datasource.driver-class-name=org.h2.Driver
-#spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+spring.datasource.url=jdbc:h2:mem:demo;DB_CLOSE_ON_EXIT=FALSE
```

Comment:
Please, pull latest changes from master.
And make sure you don't push you changes that you do to test code locally to PR.